app/src/sensors.c

```diff
@@ -18,65 +18,128 @@ LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
 
 #if ZMK_KEYMAP_HAS_SENSORS
 
-struct sensors_data_item {
-    uint8_t sensor_number;
+struct sensors_item_cfg {
+    uint8_t sensor_index;
```

Comment:
Minor: can move this to the end to maybe avoid unnecessary padding.

Reply:
Done.